elixir/apps/domain/lib/domain/actors.ex

```diff
@@ -542,6 +551,9 @@ defmodule Domain.Actors do
   def actor_disabled?(%Actor{disabled_at: nil}), do: false
   def actor_disabled?(%Actor{}), do: true
 
+  def actor_active?(%Actor{disabled_at: nil, deleted_at: nil}), do: true
```

Comment:
> or just use it directly like that in the views/business logic

Personally, I do like having the `actor_active?` method, as I feel it can make some logic and/or intent read clearer.

I may be missing something, but if I used your suggestion wouldn't I be missing the case where an actor was deleted?

Reply:
In my last push to this PR I left this as is.  If there was something I missed, please let me know.